pyproject.toml

```diff
@@ -37,6 +37,7 @@ dependencies = ["cffi==1.15.*"]
 [project.optional-dependencies]
 numpy = ["numpy==1.24.*; python_version >= '3.8'", "numpy==1.21.6; python_version == '3.7'"]
 gurobi = ["gurobipy>=8"]
+highs = ["highspy>=1.5"]
```

Comment:
I guess you must pin it to "highspy==1.5.0dev0" at the moment

Reply:
I see, so `1.5.0dev0` is considered smaller than `1.5` then.